### PR TITLE
Tidying up some code

### PR DESCRIPTION
**Pull request by Haacked**
Just tidying up some code

### CODE REVIEW

**Comment by shiftkey** [comment]
Adding this to the VS2015 milestone as it requires the end user to have access to VS2015 to build it...

**Comment by Haacked** [comment]
> it requires the end user to have access to VS2015 to build it...

It does? I tried to avoid using any new C# features. What did I do wrong?

**Comment by shiftkey** [comment]
@haacked apologies, I mixed this up with another PR - I thought I saw some `nameof` usage in here.

**Comment by Haacked** [comment]
@shiftkey nope. You might have seen my github/VisualStudio PR. Note that there's no way this would be green if I used C# 6 features, right?

**Comment by khellang** [comment]
Also, hopefully, Visual Studio would complain since `LangVersion` is set to `5` :wink:

**Comment by shiftkey** [comment]
>  Note that there's no way this would be green if I used C# 6 features, right?

CI is running against MSBuild v14, so compiler support for these new features is available. But yes, as @khellang pointed out we've locked the projects to C# 5...

**Comment by khellang** [comment]
Have you thought about when you'd like to cross over to the greener C# 6 side?

**Comment by shiftkey** [comment]
> Have you thought about when you'd like to cross over to the greener C# 6 side?

Not until after the VS2015 migration is done at the earliest...

**Comment by Haacked** [comment]
@shiftkey can this be merged?

File: Octokit/Http/RateLimit.cs [hunk 1]
```diff
@@ -52,15 +53,15 @@ public RateLimit(int limit, int remaining, long reset)
...
-        [ParameterAttribute(Key = "ignoreThisField")]
+        [Parameter(Key = "ignoreThisField")]
         public DateTimeOffset Reset { get; private set; }
 
         /// <summary>
         /// The date and time at which the current rate limit window resets - in UTC epoch seconds
         /// </summary>
-        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1811:AvoidUncalledPrivateCode")]
-        [ParameterAttribute(Key = "reset")]
-        public long ResetAsUtcEpochSeconds { get { return Reset.ToUnixTime(); } private set { Reset = value.FromUnixTime(); } }
```
Comment (shiftkey) [inline Octokit/Http/RateLimit.cs]: Yeah, I don't know why we'd need this `set`. Nice spot!

File: SolutionInfo.cs [hunk 2]
```diff
@@ -1,11 +1,12 @@
 ﻿// <auto-generated/>
+
 using System.Reflection;
 using System.Runtime.InteropServices;
 
-[assembly: AssemblyProductAttribute("Octokit")]
-[assembly: AssemblyVersionAttribute("0.14.0")]
-[assembly: AssemblyFileVersionAttribute("0.14.0")]
-[assembly: ComVisibleAttribute(false)]
+[assembly: AssemblyProduct("Octokit")]
```
Comment (shiftkey) [inline SolutionInfo.cs]: I wonder if this will be reverted the next time I run it through the FAKE build script...

**Pull request merged**